src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -216,7 +217,7 @@ public double calculateShooterHood(double distance) {
     }
 
     public void warmUp () {
-        pidController.setReference(Constants.WARM_UP_RPM, ControlType.kVelocity);
+        setShooterRpm(3000);
```

Comment:
Hmm, why not use the Constant?

Reply:
not really sure what I was thinking there, I just changed it to use the constant